nmdc/reader.go

```diff
@@ -116,7 +116,7 @@ func (r *Reader) readMsgTo(ptr *Message) error {
 		if err != nil {
 			return err
 		}
-		if bytes.ContainsAny(line, "\x00") {
+		if bytes.IndexByte(line, 0x00) != -1 {
```

Comment:
Good catch, thanks! It will actually make a difference.

Can you please make the check a bit less strict:
```suggestion
		if bytes.IndexByte(line, 0x00) >= 0 {
```

Reply:
Yes i can but i can't understand why.
!= -1 is the standard syntax used with C stdlib.
go stdlib sometimes uses != 1, sometimes >= 0:
https://golang.org/src/bytes/bytes.go?s=1959:1997#L76
https://golang.org/src/bytes/bytes.go?s=2127:2172#L81
https://golang.org/src/bytes/bytes.go?s=2299:2339#L86

it seems to me that it's just programming style.